unittests/ahbs/FV2204/IFTSTA/21035_deep.json

```diff
@@ -307,15 +482,25 @@
                 {
                   "data_element_id": "1153",
                   "discriminator": "SG15->RFF->1153",
-                  "value_pool": { "Z13": "Pr\u00fcfidentifikator" },
+                  "value_pool": [
+                    {
+                      "ahb_expression": "X",
+                      "edifact_key": "Z13",
+                      "meaning": "Pr\u00fcfidentifikator"
+                    }
+                  ],
                   "value_type": "VALUE_POOL"
                 },
                 {
                   "data_element_id": "1154",
                   "discriminator": "SG15->RFF->1154",
-                  "value_pool": {
-                    "21035": "GPKE / R\u00fcckmeldung aufLieferschein"
-                  },
+                  "value_pool": [
+                    {
+                      "ahb_expression": "X",
+                      "edifact_key": "21035",
+                      "meaning": "GPKE / R\u00fcckmeldung aufLieferschein"
```

Comment:
Sollen wir Leerzeichen und Umlaute auch anmerken?

Reply:
ne, das problem liegt in den quelldaten hier. das "ü" wird wegen `ensure_ascii=True` escaped. Gibt weniger Probleme im Git-Diff, wenn man sich nicht um encoding scheren muss.